homeassistant/components/reolink/host.py

```diff
@@ -277,6 +316,8 @@ async def renew(self) -> None:
         """Renew the subscription of motion events (lease time is 15 minutes)."""
         try:
             await self._renew(SubType.push)
```

Comment:
If they already switched to long polling should we still try to renew the webhook?  It seems like its unlikely to start working later

Reply:
Yes, there are some models like the older hardware versions of the E1 Zoom that do not publish the initial state upon subscription to the ONVIF WS base notification. For those models there is no way of knowing the subscription is working or not until the cam detect motion for the first time and a event comes in. Some users have reported that this sometimes takes many hours before someone walks past a camera....

In those cases we should use the ONVIF long polling to make sure we do receive events and it will automatically stop long polling as soon as WS base subscription kicks in and is confirmed to work.

Reply:
So we can stop the renew as soon as we get a webhook?

Reply:
Maybe store that in the config entry data so we reduce the load on the camera?

Reply:
Yes that is already implemented: when a webhook is received, self._webhook_reachable becomes True, inside _async_long_polling the long polling will be stopped and self._long_poll_task becomes None. Therefore the long polling subscription will not be renewed anymore.

Reply:
Storing in the config entry data is definitely possible, but it does add quite some complexity and I think it will result in more issues than it is going to resolve.
(Users might get stuck on long polling if the webhook subscription does not work for one time while they change network settings for instance).